web-common/src/features/dashboards/get-element-size.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Get the total size of an element, including margins
+ * @param element
+ */
+export function getEltSize(element: HTMLElement, direction: "x" | "y") {
```

Comment:
`getElmtSize` or `getElementSize` would be easier to read. I typically read "ELT" as extract-load-transform.